### DISCUSSION TITLE
Issue with Wavy Lines Appearing in Text When Using LazyVim with VSCode Integration

### DISCUSSION THREAD

**Discussion by FlatMapIO** [vscode-neovim/vscode-neovim#2344] [Q&A]
I use lazyvim and have enabled vscode in extra. In vscode-neovim, after using it for a while, this issue occurs. When I open a markdown file, there is a high probability that this problem will be triggered immediately. I have excluded many plugins in lazyvim but still do not know which plugin is causing the issue. If anyone has encountered a similar problem and resolved it, please let me know. I would greatly appreciate it.

<img width="573" alt="image" src="https://github.com/user-attachments/assets/b558bef2-e6b2-4517-9fe1-3eeee06dbe90" />

**Comment by cgjosephlee** [comment]
I have encountered same issue, do you have any solution yet?

**Comment by cgjosephlee** [comment]
Related threads here:
- https://github.com/LazyVim/LazyVim/discussions/5164
- https://github.com/LazyVim/LazyVim/issues/3692

Do `<leader>us` to disable spelling, or add following to `autocmds.lua`:
```lua
vim.api.nvim_create_autocmd("FileType", {
  group = "lazyvim_wrap_spell",
  pattern = { "text", "plaintex", "typst", "gitcommit", "markdown" },
  callback = function()
    if vim.g.vscode then
      vim.opt_local.wrap = false
      vim.opt_local.spell = false
    else
      vim.opt_local.wrap = true
      vim.opt_local.spell = true
    end
  end,
})
```

**Reply by khensunny** [reply]
or just:
```lua
if vim.g.vscode then
  vim.api.nvim_del_augroup_by_name("lazyvim_wrap_spell")
end
```
